viz.yaml

```diff
@@ -115,27 +118,113 @@ fetch:
+    id: precip_polygons_svg
+    location: cache/precip_polygons_svg.rds
     reader: rds
-    processor: geom_dataframe
+    processor: as_svg_path
+    attributes:
+      id: '{{ID}}' 
+      class: 'precip-cell'
+    depends: 
+      data: "precip_cells"
+      clip_box: "viewbox_limits"
+    scripts: [scripts/process/as_svg.R]
+  - 
+    id: state_polygons_svg
+    location: cache/state_polygons_svg.rds
+    reader: rds
+    processor: as_svg_path
+    attributes:
+      id: '{{ID}}-geom' 
+    depends: 
+      data: "state_map_data"
+      clip_box: "viewbox_limits"
+    scripts: [scripts/process/as_svg.R]
+  - 
+    id: pr_polygons_svg
+    location: cache/pr_polygons_svg.rds
+    reader: rds
+    processor: as_svg_path
+    attributes:
+      class: island
+    depends: 
+      data: "puerto_rico_map_data"
+      clip_box: "viewbox_limits"
+    scripts: [scripts/process/as_svg.R]
+  - 
+    id: state_style_svg
+    location: cache/state_style_dataframe.rds
+    reader: rds
+    processor: as_svg_use
     attributes:
       class: state
-      mouseover: hovertext('{{ID}}', evt)
-      mouseout: hovertext(' ')
+      href: '#{{ID}}-geom'
+      id: '{{ID}}'
+    depends: 
+      data: "state_map_data"
+      clip_box: "viewbox_limits"
+    scripts: [scripts/process/as_svg.R]  
+  - 
+    id: state_mouse_svg
+    location: cache/state_mouse_dataframe.rds
+    reader: rds
+    processor: as_svg_use
+    attributes:
+      onmouseover: hovertext('{{ID}}', evt)
+      onmouseout: hovertext(' ')
+      class: mouseover
+      href: '#{{ID}}-geom'
     depends: 
+      data: "state_map_data"
       clip_box: "viewbox_limits"
-      spatial_data: "state_map_data"
-    scripts: [scripts/process/geom_dataframe.R]
+    scripts: [scripts/process/as_svg.R] 
   -
-    id: precip_grid
-    location: cache/precip_grid.rds
+    id: svg_defs
+    location: cache/svg_defs.rds
+    processor: as_svg_defs
+    reader: rds
+    depends: 
+      states-geoms: state_polygons_svg
+    scripts: [scripts/process/as_svg.R] 
+  -
+    id: svg_map_style
+    location: cache/svg_map_style.rds
+    processor: as_svg_g
     reader: rds
-    cell_size: 80000
-    processor: grid_from_mask
-    scripts: [scripts/process/grid_from_mask.R]
     depends: 
-      clip_poly: "state_map_data"
+      pr-style: pr_polygons_svg
+      states-style: state_style_svg
+      precip-cells: precip_polygons_svg
+    attributes: 
+      id: map-style
+    scripts: [scripts/process/as_svg.R] 
+  -
+    id: svg_map_mouse
+    location: cache/svg_map_mouse.rds
+    processor: as_svg_g
+    reader: rds
+    depends: 
+      states-mouse: state_mouse_svg
+    attributes: 
+      id: map-mouse
+    scripts: [scripts/process/as_svg.R] 
 visualize:
+  -
+    id: svg_skeleton
+    location: cache/svg_skeleton.svg
+    visualizer: svg_skeleton
+    depends: [svg_map_style, svg_map_mouse, svg_defs]
+    scripts: ["scripts/visualize/svg_skeleton.R"]
+    reader: svg
+    mimetype: image/svg+xml
```

Comment:
In general, since this is a reusable code base, some more `comment: this thing is for this` stuff in the viz.yml seem helpful?